<ui version="4.0" >
 <class>KmChart</class>
 <widget class="QMainWindow" name="KmChart" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>267</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>182</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>PCP Chart</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="kmchart.qrc" >:/kmchart.png</iconset>
  </property>
  <property name="iconSize" >
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="chartTab" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="tabShape" >
         <enum>QTabWidget::Rounded</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <widget class="TimeButton" name="timeButton" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>53</width>
            <height>53</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>53</width>
            <height>53</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame21" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>53</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32767</width>
            <height>53</height>
           </size>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth" >
           <number>2</number>
          </property>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>2</number>
           </property>
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="TimeAxis" name="timeAxisPlot" >
               <property name="sizePolicy" >
                <sizepolicy>
                 <hsizetype>7</hsizetype>
                 <vsizetype>0</vsizetype>
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>90</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize" >
                <size>
                 <width>32767</width>
                 <height>22</height>
                </size>
               </property>
               <property name="lineWidth" >
                <number>1</number>
               </property>
               <property name="midLineWidth" >
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="dateLabel" >
               <property name="sizePolicy" >
                <sizepolicy>
                 <hsizetype>7</hsizetype>
                 <vsizetype>0</vsizetype>
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>90</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text" >
                <string>Sat Jan  1 2000 UTC</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>344</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="editChartAction" />
    <addaction name="deleteChartAction" />
    <addaction name="separator" />
    <addaction name="editTabAction" />
    <addaction name="deleteTabAction" />
    <addaction name="separator" />
    <addaction name="editSettingsAction" />
   </widget>
   <widget class="QMenu" name="Record" >
    <property name="title" >
     <string>&amp;Record</string>
    </property>
    <addaction name="recordStartAction" />
    <addaction name="recordQueryAction" />
    <addaction name="recordStopAction" />
    <addaction name="separator" />
    <addaction name="recordDetachAction" />
   </widget>
   <widget class="QMenu" name="File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileNewChartAction" />
    <addaction name="addTabAction" />
    <addaction name="closeTabAction" />
    <addaction name="separator" />
    <addaction name="fileOpenViewAction" />
    <addaction name="fileSaveViewAction" />
    <addaction name="separator" />
    <addaction name="fileExportAction" />
    <addaction name="filePrintAction" />
    <addaction name="separator" />
    <addaction name="fileQuitAction" />
   </widget>
   <widget class="QMenu" name="Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpContentsAction" />
    <addaction name="helpManualAction" />
    <addaction name="separator" />
    <addaction name="helpAboutAction" />
    <addaction name="helpSeeAlsoAction" />
    <addaction name="separator" />
    <addaction name="helpWhatsThisAction" />
   </widget>
   <widget class="QMenu" name="Time" >
    <property name="title" >
     <string>&amp;Options</string>
    </property>
    <addaction name="timeControlAction" />
    <addaction name="toolbarAction" />
    <addaction name="consoleAction" />
    <addaction name="separator" />
    <addaction name="newKmchartAction" />
   </widget>
   <addaction name="File" />
   <addaction name="Edit" />
   <addaction name="Record" />
   <addaction name="Time" />
   <addaction name="separator" />
   <addaction name="Help" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>3</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>48</width>
     <height>36</height>
    </size>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="fileNewChartAction" />
   <addaction name="addTabAction" />
   <addaction name="fileOpenViewAction" />
   <addaction name="separator" />
   <addaction name="fileExportAction" />
   <addaction name="filePrintAction" />
   <addaction name="separator" />
   <addaction name="helpWhatsThisAction" />
  </widget>
  <action name="fileOpenViewAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/view.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open View...</string>
   </property>
   <property name="iconText" >
    <string>Open View...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveViewAction" >
   <property name="text" >
    <string>&amp;Save View ...</string>
   </property>
   <property name="iconText" >
    <string>Save View ...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileExportAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/document-export.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Export...</string>
   </property>
   <property name="iconText" >
    <string>Export</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="filePrintAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/document-print.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="iconText" >
    <string>Print</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="fileQuitAction" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="iconText" >
    <string>Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="editDeleteAction" >
   <property name="text" >
    <string>&amp;Delete</string>
   </property>
   <property name="iconText" >
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="editColorsAction" >
   <property name="text" >
    <string>&amp;Colors ...</string>
   </property>
   <property name="iconText" >
    <string>Colors ...</string>
   </property>
   <property name="toolTip" >
    <string>Change Colors of Plots</string>
   </property>
  </action>
  <action name="newKmchartAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/kmchart.png</iconset>
   </property>
   <property name="text" >
    <string>New kmchart</string>
   </property>
   <property name="iconText" >
    <string>New kmchart</string>
   </property>
  </action>
  <action name="timeControlAction" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/kmtime.png</iconset>
   </property>
   <property name="text" >
    <string>Time Controls</string>
   </property>
  </action>
  <action name="toolbarAction" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Toolbar</string>
   </property>
  </action>
  <action name="consoleAction" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Console</string>
   </property>
  </action>
  <action name="editChartAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/document-properties.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Chart...</string>
   </property>
   <property name="iconText" >
    <string>Chart...</string>
   </property>
  </action>
  <action name="editTabAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/tab-edit.png</iconset>
   </property>
   <property name="iconText" >
    <string>Tab...</string>
   </property>
  </action>
  <action name="editSettingsAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/settings.png</iconset>
   </property>
   <property name="text" >
    <string>Settings...</string>
   </property>
   <property name="iconText" >
    <string>Settings...</string>
   </property>
  </action>
  <action name="closeTabAction" >
   <property name="text" >
    <string>Close Tab</string>
   </property>
   <property name="iconText" >
    <string>Close Tab</string>
   </property>
  </action>
  <action name="fileNewChartAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/document-new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New Chart...</string>
   </property>
   <property name="iconText" >
    <string>New Chart...</string>
   </property>
  </action>
  <action name="fileNewTabAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/document-new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New Chart...</string>
   </property>
   <property name="iconText" >
    <string>New Chart</string>
   </property>
  </action>
  <action name="addTabAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/tab-new.png</iconset>
   </property>
   <property name="text" >
    <string>Add Tab...</string>
   </property>
   <property name="iconText" >
    <string>Add Tab</string>
   </property>
  </action>
  <action name="deleteChartAction" >
   <property name="text" >
    <string>Delete Chart</string>
   </property>
   <property name="iconText" >
    <string>Delete Chart</string>
   </property>
  </action>
  <action name="deleteTabAction" >
   <property name="text" >
    <string>Delete Tab</string>
   </property>
   <property name="iconText" >
    <string>Delete Tab</string>
   </property>
  </action>
  <action name="helpManualAction" >
   <property name="text" >
    <string>&amp;Manual</string>
   </property>
   <property name="iconText" >
    <string>Manual</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="helpContentsAction" >
   <property name="text" >
    <string>&amp;Contents</string>
   </property>
   <property name="iconText" >
    <string>Contents</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/kmchart.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpSeeAlsoAction" >
   <property name="text" >
    <string>See Also</string>
   </property>
   <property name="iconText" >
    <string>See Also</string>
   </property>
  </action>
  <action name="helpWhatsThisAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/whatsthis.png</iconset>
   </property>
   <property name="text" >
    <string>What's This</string>
   </property>
   <property name="iconText" >
    <string>What's This</string>
   </property>
   <property name="shortcut" >
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="recordStartAction" >
   <property name="icon" >
    <iconset resource="kmchart.qrc" >:/camera-video.png</iconset>
   </property>
   <property name="text" >
    <string>Start...</string>
   </property>
  </action>
  <action name="recordQueryAction" >
   <property name="text" >
    <string>Query</string>
   </property>
  </action>
  <action name="recordStopAction" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="recordDetachAction" >
   <property name="text" >
    <string>Detach</string>
   </property>
  </action>
  <action name="actionConsole" >
   <property name="text" >
    <string>Console</string>
   </property>
  </action>
  <action name="actionToolbar" >
   <property name="text" >
    <string>Toolbar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <customwidgets>
  <customwidget>
   <class>TimeAxis</class>
   <extends>QwtPlot</extends>
   <header>timeaxis.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeButton</class>
   <extends>QToolButton</extends>
   <header>timebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kmtime.h</include>
  <include location="local" >tabdialog.h</include>
  <include location="local" >infodialog.h</include>
  <include location="local" >chartdialog.h</include>
  <include location="local" >exportdialog.h</include>
  <include location="local" >recorddialog.h</include>
  <include location="local" >settingsdialog.h</include>
  <include location="local" >qprinter.h</include>
  <include location="local" >qfiledialog.h</include>
  <include location="local" >qassistantclient.h</include>
  <include location="local" >timeaxis.h</include>
  <include location="local" >timebutton.h</include>
  <include location="local" >timeaxis.h</include>
 </includes>
 <resources>
  <include location="kmchart.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>fileNewChartAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>fileNewChart()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileOpenViewAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>fileOpenView()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveViewAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>fileSaveView()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileExportAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>fileExport()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filePrintAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>filePrint()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileQuitAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>fileQuit()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpContentsAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>helpContents()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpAboutAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>helpAbout()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpSeeAlsoAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>helpSeeAlso()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpWhatsThisAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>whatsThis()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeControlAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>optionsTimeControl()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolbarAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>optionsToolbar()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>consoleAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>optionsConsole()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newKmchartAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>optionsNewKmchart()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeTabAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>closeTab()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSettingsAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>editSettings()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chartTab</sender>
   <signal>currentChanged(QWidget*)</signal>
   <receiver>KmChart</receiver>
   <slot>activeTabChanged(QWidget*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeButton</sender>
   <signal>clicked()</signal>
   <receiver>KmChart</receiver>
   <slot>optionsTimeControl()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editChartAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>editChart()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteChartAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>deleteChart()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editTabAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>editTab()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteTabAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>closeTab()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addTabAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>addTab()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpManualAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>helpManual()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordStartAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>recordStart()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordQueryAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>recordQuery()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordStopAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>recordStop()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordDetachAction</sender>
   <signal>activated()</signal>
   <receiver>KmChart</receiver>
   <slot>recordDetach()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
